src/transfer_raw_zip.py

```diff
@@ -711,7 +726,9 @@ def initialize():
         logger.warning("dry_run=True, no writes")
 
     source_butler = Butler(config.fromrepo, skymap="lsst_cells_v1")
-    dest_butler = Butler(config.torepo, writeable=True)
+    for torepo in config.torepos:
+        dest_butler = Butler(torepo, writeable=True)
+        dest_butlers.append(dest_butler)
```

Comment:
More compact is:
```suggestion
    dest_butlers = [Butler(torepo, writeable=True) for torepo in config.torepos]
```
This should also solve the immediate test failure.